packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -861,6 +860,6 @@ curl -v -X GET \
 "https://${yourOktaDomain}/api/v1/logs?since=2017-10-01T00:00:00.000Z"
 ```
 
-and retrieve the next page of events through the [`link` response header](/docs/reference/api-overview/#link-header) value with the `next` link relation. Continue this process until no events are returned.
+Then retrieve the next page of events through the [`link` response header](/docs/reference/api-overview/#link-header) value with the `next` link relation. Continue this process until no events are returned.
```

Comment:
Remove code format for `link` in the reference; it doesn't render great on the page, imo.